ddtrace/contrib/dbapi/__init__.py

```diff
@@ -178,6 +178,46 @@ def __init__(self, conn, pin=None, cfg=None, cursor_cls=None):
         self._self_cursor_cls = cursor_cls
         self._self_config = cfg or config.dbapi2
 
+    def __enter__(self):
+        """Context management is not defined by the dbapi spec.
+
+        This means unfortunately that the database clients each define their own
+        implementations.
+
+        The ones we know about are:
+
+        - mysqlclient<2.0 which returns a cursor instance. >=2.0 returns a
```

Comment:
We have others to add here no?

Reply:
Ah yes we have since looked into others, I'll update